### ISSUE TITLE
Hi, can i train a binary problem?

### ISSUE THREAD

**Issue by Angel113110** [siqueira-hc/Efficient-Facial-Feature-Learning-with-Wide-Ensemble-based-Convolutional-Neural-Networks#11]
Dear author, thanks for sharing so wonderful work. I have a question to ask you. In my case, I only focus on smile expression. Can I train a binary problem based on your work? Can you give some help or advice? Thanks!

**Comment by siqueira-hc** [comment]
Dear Angel113110,

Thank you very much for your message and your interest.

There are several ways to use our model in this case, for example:

1. Since smiling is strongly related to happy facial expressions in public datasets, you could directly use our model for facial expression recognition and post-processing happy classifications in order to detect smiles.

2. The last layer of ESR-9 uploaded in this repository has two neurons (units) responsible for continuous affect perception. You could re-train this layer for your binary problem on your dataset using the cross-entropy loss function.

3. The second last layer of ESR-9 uploaded in this repository has eight neurons (units) responsible for discrete emotion perception. You could use them as features to train another model for smile detection.

Best,
Henrique Siqueira

**Comment by Angel113110** [comment]
Dear author. If I want to use the second way to train a binary task. How to do with it? I'm new in FER field. Can you say more details?
